util/rtconf-test/Makefile

```diff
@@ -0,0 +1,29 @@
+SOURCE_DIR=../../src/mainboard/pcengines/apu2
```

Comment:
We can define something like $(top) variable @BartekLew .
This is commonly used in coreboot Makefiles. These dots and slashes do not look safe.